README.md

```diff
@@ -57,22 +57,22 @@ By the end of the course, you will be able to ...
+| 10 | Wed, Sep 11 | -                             | [Code Quality, Reuse & Error Handling] | Jess Dahmen | Alan Davis |
+| 12 | Thu, Sep 12 | Intro to Flask <br/> Flask Templating | [Testing RESTful Routes] | Dani Roxberry | Meredith Murphy |
+| 11 | Mon, Sep 16 | Pseudocode & Flowchart Diagrams <br/> Problem Solving Strategies <br/> Code Quality, Reuse & Error Handling | [Debugging & Testing] | Jess Dahmen | Alan Davis |
+| 14 | Tue, Sep 17 | URLs, HTTP, REST, & Reading Errors <br/> Testing RESTful Routes | [Document-based Databases] | Dani Roxberry | Meredith Murphy |
+| 13 | Wed, Sep 18 | -                             | [OOP Part 1: Encapsulation] | Jess Dahmen | Alan Davis |
 | 15 | Thu, Sep 19 | -                             | [ERDs, Resource Associations, & MongoDB] | Dani Roxberry | Meredith Murphy |
 | 16 | Mon, Sep 23 | Debugging & Testing <br/> OOP 1 | [OOP Part 2: Inheritance] | Jess Dahmen | Alan Davis |
-| 17 | Tue, Sep 24 | -                             | [OOP Part 3: Polymorphism] | Jess Dahmen | Alan Davis |
-| 18 | Wed, Sep 25 | Document-based Databases <br/> ERDs, Resource Associations, & MongoDB | [Project Planning & User-Centered Development]| Jess Dahmen | Alan Davis |
-| 19 | Thu, Sep 26 | OOP 2 <br/> OOP 3             | [Herd Immunity Simulation] | Jess Dahmen | Alan Davis |
-| 20 | Mon, Sep 30 | -                             | [The DOM & Bootstrap] | Dani Roxberry | Meredith Murphy |
-| 21 | Tue, Oct 1  | Project Planning & User-Centered Development <br/> The DOM & Bootstrap | Contractor Project Lab Day | Dani Roxberry | Meredith Murphy |
+| 20 | Tue, Sep 24 | -                             | [The DOM & Bootstrap] | Dani Roxberry | Meredith Murphy |
+| 17 | Wed, Sep 25 | -                             | [OOP Part 3: Polymorphism] | Jess Dahmen | Alan Davis |
+| 18 | Thu, Sep 26 | Document-based Databases <br/> ERDs, Resource Associations, & MongoDB | [Project Planning & User-Centered Development]| Jess Dahmen | Alan Davis |
+| 19 | Mon, Sep 30 | OOP 2 <br/> OOP 3             | [Herd Immunity Simulation] | Jess Dahmen | Alan Davis |
+| 21 | Tue, Oct 1  | Project Planning & User-Centered Development <br/> The DOM & Bootstrap | Contractor Project Lab Day | Both | Both |
 | 22 | Wed, Oct 2  | -                             | [RESTful APIs & Deployment Environments] | Dani Roxberry | Meredith Murphy |
 | 23 | Thu, Oct 3  | -                             | [Computer Components & Organization] | Jess Dahmen | Alan Davis |
-| 24 | Mon, Oct 7  | -                             | Herd Immunity Lab | Jess Dahmen | Alan Davis |
+| 24 | Mon, Oct 7  | -                             | Herd Immunity Lab | Both | Both |
 | 25 | Tue, Oct 8  | -                             | [CPU Instructions & Memory] | Jess Dahmen | Alan Davis |
 | 26 | Wed, Oct 9  | RESTful APIs & Deployment Environments <br/> Computer Components & Organization <br/> CPU Instructions & Memory | Project Lab Day | Jess Dahmen | Alan Davis |
```

Comment:
two things:

1) feel like "Both" should be here since it's an open lab day, so would make sense to have one CS and one BEW instructor in each lab

2) mixing BEW and CS quiz topics here. Fine if everyone is ok with that, but I know there were reservations before on it from instructors. But if it's fine with y'all, then no biggie